src/paths/user/account.yaml

```
@@ -0,0 +1,19 @@
+get:
+  tags:
+    - user
+  description: "Show the account details of the signed-in user"
+  summary: "/<user_id>/account"
+  parameters:
+    - $ref: "../../../parameters/path/user_id.yaml"
+
+
+
+  responses:
+    200:
+      description: "Account details of the signed-in user"
```

```suggestion
      description: "Account details of a user"
```
The user specified might not be the one doing the request.